### DISCUSSION TITLE
[question]我想要取消这个CSS样式：.ant-tabs-tabpane的` padding-left: 24px;`  请问在使用antd/Tabs的时候，应该如何取消它呢？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [ant-design/ant-design#51840] [Q&A] [answered]
您好，基于：Tabs
https://ant-design.antgroup.com/components/tabs-cn 

![image](https://github.com/user-attachments/assets/b2f38b62-611d-433a-b817-41a4e9712e77)

这里有`Tab position:left`的样式：

我们使用浏览器控制台Element可看到` id="rc-tabs-1-panel-1"`的div：

```
<div id="rc-tabs-1-panel-1" role="tabpanel" tabindex="0" aria-labelledby="rc-tabs-1-tab-1" aria-hidden="false" class="ant-tabs-tabpane ant-tabs-tabpane-active">...</div>
```
有一个样式：
```
:where(.css-dev-only-do-not-override-1x0dypw).ant-tabs-left >.ant-tabs-content-holder >.ant-tabs-content>.ant-tabs-tabpane, :where(.css-dev-only-do-not-override-1x0dypw).ant-tabs-left >div>.ant-tabs-content-holder >.ant-tabs-content>.ant-tabs-tabpane {
    padding-left: 24px;
}
```

我想要取消这个` padding-left: 24px;`

也就是说蓝色框这个gap，想要没有padding：
![image](https://github.com/user-attachments/assets/0df623ea-199e-48c5-b146-f6f42ca51506)

请问在使用antd/Tabs的时候，应该如何取消它呢？

**Comment by aojunhao123** [comment, marked as answer]
可以通过ConfigProvider去覆写组件内部的样式：
```tsx
    <ConfigProvider tabs={{ className: "my-tabs" }}>
      <Tabs
        tabPosition={tabPosition}
        items={new Array(3).fill(null).map((_, i) => {
          const id = String(i + 1);
          return {
            label: `Tab ${id}`,
            key: id,
            children: `Content of Tab ${id}`,
          };
        })}
      />
    </ConfigProvider>
```
![QQ_1732971310190](https://github.com/user-attachments/assets/d423df16-76db-46a9-83f6-3b0c934da971)

**Comment by aojunhao123** [comment]
如果想了解主题拓展更细节的方面，可以看官方的这篇博客：https://ant-design.antgroup.com/docs/blog/config-provider-style-cn
